### PR TITLE
fix: big update

### LINKED ISSUE #5: Integrate with schema utils js
https://github.com/open-rpc/schema-utils-js

### PR DESCRIPTION
**Pull request by BelfordZ**
fixes #5

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { reduce } = require('lodash');
```
Comment (BelfordZ) [inline src/server.js]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "description": "Provides a mock RPC API given an open rpc schema",
   "main": "server.js",
   "dependencies": {
+    "@open-rpc/schema-utils-js": "^1.0.1",
```
Comment (BelfordZ) [inline package.json]:
```suggestion
    "@open-rpc/schema-utils-js": "^1.0.1",
    "jayson": "^2.1.1",
    "json-schema-faker": "^0.5.0-rc16",
    "json-schema-ref-parser": "^6.0.3"
  },
```

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
...
     "jayson": "^2.1.1",
     "json-schema-faker": "^0.5.0-rc16",
     "json-schema-ref-parser": "^6.0.3",
```
Comment (BelfordZ) [inline package.json]: remove lodash here

Comment (BelfordZ) [inline package.json]: remove this aswell

File: package.json [hunk 4]
```diff
@@ -4,10 +4,9 @@
   "description": "Provides a mock RPC API given an open rpc schema",
   "main": "server.js",
   "dependencies": {
+    "@open-rpc/schema-utils-js": "^1.0.1",
     "jayson": "^2.1.1",
-    "json-schema-faker": "^0.5.0-rc16",
-    "json-schema-ref-parser": "^6.0.3",
-    "lodash": "^4.17.11"
+    "json-schema-faker": "^0.5.0-rc16"
```
Comment (BelfordZ) [inline package.json]: should look into better / stable release of this thing

File: bin/cli.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+const program = require('commander');
+const server = require('../src/server');
+
+program
+  .version(require('./get-version'))
+  .option('-s, --schema [schema]', 'JSON string or a Path/Url pointing to an open rpc schema')
```
Comment (shanejonas) [inline bin/cli.js]:
```suggestion
  .option('-s, --schema [schema]', 'JSON string or a Path/Url pointing to an OpenRPC Schema')
```

**Pull request merged**